<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>820</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{

max-width: 1280px;
min-width:1280px;
max-height: 820px;
min-height:820px;
background-color:#013763;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>911</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

font-size:40px;
font-weight: bold;
color : #DBD9D9;


}</string>
    </property>
    <property name="text">
     <string>CoreTech Innovations - IMT School </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

color : #DBD9D9;
font-size: 32px;
font-weight:bold;
width : 452px;
height: 39px;

}</string>
    </property>
    <property name="text">
     <string>Welcome To The Bank System</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

font-size: 24px;
color : #DBD9D9;
width: 328px;
height: 29px;


}</string>
    </property>
    <property name="text">
     <string>Developed by: Eng. Safy Srio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbContinue">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>550</y>
      <width>187</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

border: none;
background-color: #DBD9D9;
border-radius:16px;
color:#013763;
min-width:187px;
min-height:65px;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>Connect-&gt;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>4</width>
      <height>481</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>125</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>4</width>
      <height>1012</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView{

border-radius:2px;

}</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>240</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_CoreTechIcon">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ImtIcon">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
